assets/i18n/en.json

```diff
@@ -711,9 +737,15 @@
     "I18N_LEARNER_GROUP_SYLLABUS_ITEM_NOT_STARTED_YET": "Not started yet",
     "I18N_LEARNER_GROUP_SYLLABUS_TAB": "Syllabus",
     "I18N_LEARNER_GROUP_VIEW_DETAILS": "View Details",
+    "I18N_LEARNER_GROUP_VIEW_OVERVIEW_SUMMARY_TITLE": "Your overall progress through syllabus",
```

Comment:
This reads weirdly, I don't think it's grammatically correct.

I'm getting a bit worried about this. Have the strings been approved by the UX Writing team? /cc @dchen97

Reply:
I am not sure, I took them from figma mocks and Mindi did say that the design team was in contact wth ux team.

Reply:
Explore your progress through your lessons:

Reply:
done